src/logger_dispatch_overrides.jl

```diff
@@ -80,12 +89,31 @@ struct TBImage <: WrapperLogType
     data::AbstractArray
     format::ImageFormat
 end
+"""
+    TBImage(image)
+
+Renders `image` to PNG and logs it as an image to TensorBoard.
+`showable("image/png", image)` must be true.
+"""
+TBImage(image) = TBImage(image, PngImage)
```

Comment:
I suppose it doesn't handle non-array object that is showable as image/png?

Reply:
Didn't I relax the type constraint on `TBImage`? 
No I did not.
-_-. I definitely should not write code anymore late at night in a night train

Reply:
Oh, good to know that it is/was in your plan.

Reply:
done